python/lsst/cell_coadds/_identifiers.py

```diff
@@ -54,7 +54,7 @@ def __getitem__(self, key: str) -> Any:
         return getattr(self, key)
 
     @classmethod
-    def from_data_id(cls, *args, **kwargs) -> Self:
+    def from_data_id(cls, *args, **kwargs) -> Self:  # type: ignore
```

Comment:
If it's just taking `*args, **kwargs` and raising I don't think the base implementation is doing any good; subclass signatures are simply different, and that means it's not a base-class interface.

Reply:
Are you suggesting we remove this classmethod altogether? How do we still signal that all subclasses must still implement a constructor from `data_id`?